gce/api-servers.tf

```diff
@@ -27,13 +27,37 @@ resource "google_compute_http_health_check" "api" {
   healthy_threshold = "${var.health_check_healthy}"
   unhealthy_threshold = "${var.health_check_unhealthy}"
 }
+resource "google_compute_http_health_check" "api_https" {
+  name = "${var.env}-tsuru-api-https"
+  port = 443
+  request_path = "/info"
+  check_interval_sec = "${var.health_check_interval}"
+  timeout_sec = "${var.health_check_timeout}"
+  healthy_threshold = "${var.health_check_healthy}"
+  unhealthy_threshold = "${var.health_check_unhealthy}"
+}
 resource "google_compute_target_pool" "api" {
   name = "${var.env}-tsuru-api-lb"
   instances = [ "${google_compute_instance.api.*.self_link}" ]
   health_checks = [ "${google_compute_http_health_check.api.name}" ]
```

Comment:
I think it'd be preferable to have a single (and same type of) health check for both pools. But I guess this'll get tidied up as part of disabling HTTP:8080?